UM/Message.py

```diff
@@ -38,14 +43,17 @@ def __init__(self, text: str = "", lifetime: int = 30, dismissable: bool = True,
         :param text: Text that needs to be displayed in the message
         :param lifetime: How long should the message be displayed (in seconds).
             if lifetime is 0, it will never automatically be destroyed.
-        :param dismissible: Can the user dismiss the message?
+        :param dismissable: Can the user dismiss the message?
         :param title: Phrase that will be shown above the message.
         :param image_source: an absolute path where an image can be found to be
         displayed (QUrl.toLocalFile()) can be used for that.
         :param image_caption: Text to be displayed below the image (or anywhere
         really, it's up to the QML to handle that).
         :param progress: Is there any progress to be displayed? if -1, it's seen
         as indeterminate.
+        :param message_type: Defines the type of message (POSITIVE, NEUTRAL, WARNING, ERROR, default: NEUTRAL).
+                            According to the type, an icon appears next to the message title. The NEUTRAL messages
+                            contain no icon.
```

Comment:
Indentation is inconsistent with the rest of the params here.

Reply:
Should we document where to get this from? In the type it just says "int", and this documentation doesn't mention that it should be a member of the `MessageType` class.

Reply:
Fixed in 6366138fdc2e7672e14a95b8559bffccba197039.